src/main/java/seedu/duke/Ui.java

```diff
@@ -0,0 +1,104 @@
+package seedu.duke;
+
+import java.text.DecimalFormat;
+import java.lang.System;
+
+public class Ui {
+    public static final String ANSI_RESET = "\u001B[0m";
+    public static final String ANSI_BLACK = "\u001B[30m";
+    public static final String ANSI_RED = "\u001B[31m";
+    public static final String ANSI_GREEN = "\u001B[32m";
+    public static final String ANSI_YELLOW = "\u001B[33m";
+    public static final String ANSI_BLUE = "\u001B[34m";
+    public static final String ANSI_PURPLE = "\u001B[35m";
+    public static final String ANSI_CYAN = "\u001B[36m";
+    public static final String ANSI_WHITE = "\u001B[37m";
+    public static final String ANSI_GRAY = "\u001B[90m";
+    public static final String FULL_BLOCK = "█";
+    public static final String HALF_BLOCK = "▌";
+    public static final int MAX_BAR = 16;
+    public static final int BAR_WIDTH = 10;
+    public static final String SPACE = " ";
+    public static final String EMOJI_1 = " ᕦ(ò_óˇ)";
+    public static final String FITBOT_V0 = "  ______ _ _   _           _"
+            + System.lineSeparator()
+            + " |  ____(_) | | |         | |"
+            + System.lineSeparator()
+            + " | |__   _| |_| |__   ___ | |_"
+            + System.lineSeparator()
+            + " |  __| | | __| '_ \\ / _ \\| __|"
+            + System.lineSeparator()
+            + " | |    | | |_| |_) | (_) | |_"
+            + System.lineSeparator()
+            + " |_|    |_|\\__|_.__/ \\___/ \\__|";
+
+    public static void printCalorieProgress(int calorieConsumed, int calorieGoal) {
+        if (calorieGoal > 0) {
+            float percentage = (float) (((float) calorieConsumed / calorieGoal) * 100.0);
+            String newPercentage = getPercentage(percentage);
+            int barNum = getBarNum(percentage);
+            String result = getResult(barNum);
+            String color = determineColor(percentage);
+            System.out.println(color + result + newPercentage + '%' + ANSI_RESET);
+        }
+    }
+
+    private static String getPercentage(float percentage) {
```

Comment:
Woah the progress bar is very cool! But perhaps the methods can be named better, e.g. `getPercentage()` -> `getPercentageForCalorieProgress()`, as it is hard to tell now from the names what they are meant to do.